include/splinterdb/splinterdb.h

```diff
@@ -32,12 +53,18 @@ typedef struct {
    // For a simple reference implementation, see default_data_config.h
    data_config *data_cfg;
 
-
    // optional advanced config below
    // if unset, defaults will be used
    void *heap_handle;
    void *heap_id;
 
+   // Shared memory support
+   uint64 shmem_size;
+   _Bool  use_shmem; // Default is FALSE.
+   _Bool  trace_shmem_allocs;
+   _Bool  trace_shmem_frees;
+   _Bool  trace_shmem; // Trace both allocs & frees from shared memory
```

Comment:
Ideally, we should have one `_Bool trace_flags` bitmap, where each feature needing tracing turns ON its bit.

I didn't go that far, yet, ... unsure how the team feels about this extra tracing baggage.
I did find these traceflags very useful during dev & stabilization, so would like to retain these.

Reply:
Re-discussed with @rtjohnso -- 

Make these global bool variables in shmem.c and take them out of this public interface `struct splinterdb_config{}`

In config_parse(), when the `--trace` flag is supplied, toggle these boolean variables.